components/recurring-contributions/RecurringContributionsCard.js

```diff
@@ -90,46 +85,33 @@ const RecurringContributionsCard = ({
                 <P fontSize="12px" lineHeight="20px" color="black.700">
                   (
                   <FormattedMoneyAmount
-                    amount={contribution.amount.value * 100}
+                    amount={contribution.amount.valueInCents - contribution.platformContributionAmount.valueInCents}
                     currency={contribution.amount.currency}
                     showCurrencyCode={false}
-                    precision={0}
+                    precision="auto"
```

Comment:
We want to avoid truncating the number, this was showing:

> $17.25 USD / month
> ($15 + $2)

`precision="auto"` is a good fit for that, it hides the `,00` if there's no decimals but show them if there's any.